Framework/src/CubismMotionSync.cpp

```diff
@@ -248,6 +248,7 @@ CubismMotionSync::CubismMotionSync(CubismModel* model, CubismMotionSyncData *dat
     {
         _processorInfoList.PushBack(CubismProcessorInfo(processorList[i], model, data->GetSetting(i)));
```

Comment:
Additional Note:
The reason why `CSM_NEW` and `CSM_DELETE` are not executed for analysisResult in the constructor and destructor of CubismProcessorInfo is that the value addition process to this `csmVector<CubismProcessorInfo> _processorInfoList;` is a copy operation, and the constructor and destructor are executed on the copy source.
By changing `csmVector<CubismProcessorInfo> _processorInfoList;` to `csmVector<CubismProcessorInfo*> _processorInfoList;`, the above issue can be avoided. However, this was not done in this instance because it would increase the code differences and change the design of object lifetime management.

---
追記：
なぜ、CubismProcessorInfoのコンストラクタ、デストラクタ内で`analysisResult`に対する `CSM_NEW`, `CSM_DELETE` を実行しないのかというと、この `csmVector<CubismProcessorInfo> _processorInfoList;` への値追加処理がコピー処理であり、コピー元でコンストラクタ、デストラクタが実行される（コピー元、コピー先で`analysisResult`のポインタは共有されて単一である）からです。
 `csmVector<CubismProcessorInfo> _processorInfoList;` を、`csmVector<CubismProcessorInfo*> _processorInfoList;` に変更することで、上記の課題を回避することが出来ますが、コード差分が増えることから今回は避けました。

Reply:
Thank you for your confirmation.

We have also reviewed it internally and understand the intent of the modification. We consider this change to be acceptable as we further evaluate the use of csmVector<CubismProcessorInfo*> going forward.

---

ご確認ありがとうございます。

弊社内でも確認し、修正意図について承知いたしました。
今後 `csmVector<CubismProcessorInfo*>` に検討するうえで、この修正を問題なしとします。